safe_cli/operators/safe_operator.py

```diff
@@ -430,6 +435,43 @@ def approve_hash(self, hash_to_approve: HexBytes, sender: str) -> bool:
                     )
                     return False
 
+    def sign_message(
+        self,
+        eip191_message: Optional[str] = None,
+        eip712_message_path: Optional[str] = None,
+    ) -> bool:
+        if eip712_message_path:
+            try:
+                message = json.load(open(eip712_message_path, "r"))
+                message_bytes = b"".join(eip712_encode(message))
```

Comment:
Should I sign the hash of EIP712 or the string hash following EIP191? I'm asking because is the way that the transaction service accepts.